site/src/playground.njk

```diff
@@ -31,7 +31,7 @@
       <span class="center">Loading CSS editor…</span>
     </div>
     <script type="module" src="/js/runtime.bundle.js"></script>
-    <script type="module" src="/js/628.bundle.js"></script>      
+    <script type="module" src="/js/932.bundle.js"></script>
```

Comment:
We could use the webpack manifest plugin to update this automatically, but it does not change that often (it changed when bumping webpack from 5.70 to 5.71)